app/(root)/case-studies/[id]/page.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+type Props = {};
```

Comment:
I'm assuming you're going to use this later. Otherwise, you should remove this.

Reply:
Resolved